ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -78,6 +78,11 @@ sporkaddr={{ dashd_sporkaddr }}
 sporkkey={{ dashd_sporkkey }}
 {% endif %}
 
+{% if dashd_walletnode is not none and coinjoin_wallet_count is defined %}
+coinjoinmultisession=1
```

Comment:
Should we add `coinjoinautostart=1` here too?

Reply:
Would rather not, let's start it manually when we run stage2 so we can control it. We may have a network with coinjoin wallet count that we don't want to start yet